scheduler/src/cook/components.clj

```diff
@@ -557,6 +561,17 @@
                    (merge {:interval-seconds 300
                            :dru-scale 1.0}
                           rebalancer))
+                       
+     :optimizer (fnk [[:config {optimizer nil}]]
+                     ;; TODO include a datomic connection to config so we can write purchase decisions
+                     (let [optimizer-config
+                           (merge {:host-feed {:create-fn 'cook.mesos.optimizer/create-dummy-host-feed
+                                               :config {}}
+                                   :optimizer {:create-fn 'cook.mesos.optimizer/create-dummy-optimizer
+                                               :config {}}
+                                   :optimizer-interval-seconds 30}
```

Comment:
Where is `optimizer-interval-seconds` used?

Reply:
That.. is a great question. It currently isn't. Will fix.

Reply:
in mesos.clj make-trigger-chans